### DISCUSSION TITLE
YARP tutorials in Python

### DISCUSSION THREAD

**Discussion by ddetommaso** [robotology/community#541] [Show and tell]
Dear all,

I want to share with the community a bunch of basic tutorials we developed recently in @s4hri for Python developers who are just starting out with YARP!

https://github.com/s4hri/yarp-python-tutorials

Thanks to @Nicolasr3 for the great work!

**Comment by pattacini** [comment]
Great! Thanks for sharing @ddetommaso 👍🏻 
Just mentioning @robotology/everyone for spreading the news!
